api/create-data/models/create-data.settings.json

```diff
@@ -0,0 +1,57 @@
+{
+  "kind": "collectionType",
+  "collectionName": "create_data_s",
+  "info": {
+    "name": "createData ",
+    "description": ""
+  },
+  "options": {
+    "increments": true,
+    "timestamps": true,
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "identity": {
```

Comment:
what is identify? Is this supposed to be the id field that's in the createData.json ?